index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+  </head>
+  <body>
+    <header id="header">
+      <nav id="nav-bar" class="header__nav-bar">
+        <a href="#header" class="nav-link">Hard Rock</a>
+        <a href="#email" class="nav-link">mmmm@Mverse</a>
+        <a href="#submit" class="nav-link">Submit Form</a>
+      </nav>
+      <img
+        id="header-img"
+        src="./src/assets/image/1995-porsche-911-carrera-archived-road-test-review-car-and-driver-photo-545611-s-original.jpg"
+        width="50%"
+        alt="a picture of a lightBlue porshe car"
+      />
+    </header>
+    <iframe
+      id="video"
+      width="100%"
+      height="315"
+      src="https://www.youtube.com/embed/e1bl7Db0DN0"
+      title="YouTube video player"
+      frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+      allowfullscreen
+    ></iframe>
+    <form
+      action="https://www.freecodecamp.com/email-submit
+    "
+      id="form"
+    >
+      <input
+        id="email"
+        name="email"
+        type="email"
+        placeholder="Here goes your email..."
+        required
+      />
+      <input type="submit" id="submit" />
+    </form>
```

Comment:
It would be a good idea as well as for Semantic reasons, to place the above code within a `main` tag. Also if the `<iframe>` is not related to your `<form>` element, consider moving each element within its own `<section>`. 👍🏼